templates/kafka-connect/kafka-connect-s2i.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kafka.strimzi.io/v1alpha1
```

Comment:
This file is changed one more time. Get the updated version from `master` branch of the `strimzi-kafka-operator` project.
But I have a question on this one. It's not an OpenShift template, why it's under `templates` folder?